code/game/machinery/computer/camera_advanced.dm

```diff
@@ -70,14 +70,25 @@
 	if(M == current_user)
 		remove_eye_control(M)
 
+/obj/machinery/computer/camera_advanced/proc/can_use(mob/living/user)
+	return TRUE
+
+/obj/machinery/computer/camera_advanced/can_use(mob/user)
+	if(!isabductor(user))
+		return FALSE
+	return ..()
```

Comment:
Don't define the same proc twice for the same typepath, also this makes all advanced cameras only usable by abductors when we have non abductor advanced cameras.

Reply:
Yeah that was a mistake I accidently wiped out the abductor part of the typepath sigh